nutrition_labels/tech_grant_tagger.py

```diff
@@ -73,31 +50,49 @@ def clean_grants_data(self, previous_grant_data):
                 ).tolist()
         return grant_data
 
-    def load_grants_text(self):
+    def load_grants_text(self, grants_data_path):
         """
         Field names need an update here!
         """
-        grant_data = pd.read_csv(self.input_path)
+        grant_data = pd.read_csv(grants_data_path)
         grant_data = self.clean_grants_data(grant_data)
         grants_text = grant_data['Grant texts'].tolist()
         self.grants_ids = grant_data[self.grant_id_col].tolist()
 
         return grants_text
 
+    def load_model(self, model_path):
+
+        # Loading a trained model and vectorizer to predict on all the grants data:
+        grant_tagger_loaded = GrantTagger()
```

Comment:
careful here, my project is also called grants_tagger :p

Reply:
how is `GrantTagger` different from `TechGrantModel`, why there are two classes?

Reply:
yes, good point. GrantTagger is training single models (e.g. Count+SVM). TechGrantModel is combining the trained models to make ensemble predictions.

Reply:
maybe TechGrantModel-> EnsembleGrantTagger?

Reply:
I see, yeah that is a better name imo